EIPS/eip-5380.md

```diff
@@ -0,0 +1,91 @@
+---
+eip: 5380
+title: EIP-721 Entitlement Extension
+description: Allows token owners to grant the ability for others to use specific properties of those tokens
+author: Pandapip1 (@Pandapip1), Tim Daubenschütz (@TimDaub)
+discussions-to: https://ethereum-magicians.org/t/pr-5380-eip-4907-alternative-design/10190
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-03-11
+requires: 165, 721
+---
+
+## Abstract
+
+This EIP proposes extensions of the commonly-used [EIP-721](./eip-721.md) standard that allows token owners to grant limited usage of those tokens to other addresses.
+
+## Motivation
+
+There are many scenarios in which it makes sense for the owner of a token to grant certain properties to another address. One use case is renting tokens. If the token in question represents a trading card in an on-chain TCG, one might want to be able to use that card in the game without having to actually buy it. Therefore, the renter might grant the person renting the card the "property" of it being able to be played in the TCG. However, this property should only be able to be assigned to one person at a time, otherwise a contract could simply "rent" the card to everybody. If the token represents usage rights instead, the property of being allowed to use the associated media does not need such a restriction, and there is no reason that the property should be as scarce as the token. 
+
+## Specification
+
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY" and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+### `IERC5380`
+
+```solidity
+/// SPDX-License-Identifier: CC0-1.0
+pragma solidity ^0.8.0;
+
+interface IERC5380 is IERC165 {
+    /// @notice Emitted when the amount of entitlement a user changes. If user is the zero address, then the user is the owner
+    event Entitle(address indexed user, address indexed contract, uint256 indexed tokenId);
+
+    /// @notice             Set the user associated with the given EIP-721 token as long as the owner is msg.sender.
+    /// @dev                SHOULD NOT revert if the owner is not msg.sender.
+    /// @param  user        The user to grant the entitlement to
+    /// @param  contract    The property to grant
+    /// @param  tokenId     The tokenId to grant the properties of
+    function entitle(address user, address contract, uint256 tokenId) external;
```

Comment:
I suggest add a `bytes calldata data` for future extension or reason of entitlement

Reply:
I'll add an additional function that supports it. I'm not a huge fan but I can see possible use cases.